pkg/resourceinterpreter/predefinedconfigurableinterpreter/configurable.go

```diff
@@ -0,0 +1,221 @@
+package predefinedconfigurableinterpreter
```

Comment:
The implementation of `predefinedconfigurableinterpreter` is basicly the same as `configurableinterpreter`. It will import some config files of third-party resources. Logically it belongs to `configurableinterpreter` and the current implementation has a lot of redundant code.

Reply:
/cc @XiShanYongYe-Chang for opinions.

Reply:
Yes, the core idea of `predefinedconfigurableinterpreter` is to be consistent with `configurableinterpreter` and reduce user learning costs. Therefore, I moved the code for `luavm` and `accessor` from `configurableinterpreter` to `configurable`, in order to reuse the code.

Reply:
> the current implementation has a lot of redundant code.

I don't want the `predefinedconfigurableinterpreter` and `configurableinterpreter` to be too tightly coupled, so this code is necessary.